package day38_exceptions;

public class self {
    public static void main(String[] args) throws InterruptedException { // Exception
        String str = null;

        Thread.sleep(5000);

        System.out.println("End");
    }
}
